lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -181,31 +181,30 @@ def self.get_token
         UI.crash!("Wrong value for FIREBASE_TOKEN")
       end
 
-      def self.get_app(params)
+      def self.validate_app!(params)
```

Comment:
nit: Consider just passing the `app_id` since it's the only value in the `params` hash that is used.